resources/js/Pages/Auth/Signup.vue

```diff
@@ -0,0 +1,67 @@
+
+<script setup>
+import { useForm } from '@inertiajs/vue3'
+
+const form = useForm({
+  name: '',
+  email: '',
+  password: '',
+  errors: {},
+})
+
+const submit = async () => {
+  try { 
+    form.post('/register') 
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+</script>
+
+<template>
+  <div class="flex h-screen w-full items-center justify-center bg-gradient-to-br from-blue-600 to-indigo-600">
+    <form @submit.prevent="submit">
+      <div class="w-screen max-w-sm rounded-xl bg-white px-10 py-8 shadow-md">
+        <div class="space-y-4">
+          <div class="flex items-center justify-between">
+            <h1 class="text-center text-2xl font-semibold text-gray-600">
+              Register
+            </h1>
+            <button type="button" class="mr-2 inline-flex items-center rounded-lg bg-blue-700 p-2.5 text-center text-sm font-medium text-white hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">
+              <a href="/">back </a>
+              <svg aria-hidden="true" class="h-5 w-5" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Instead of using svg, we can use icon here. I'd recommend heroicons. But the design will change as you said, so it's just a piece of advise for the future.